ross/api_report

```diff
@@ -0,0 +1,538 @@
+import numpy as np
+from scipy.signal import argrelextrema
+
+from rotor_assembly import Rotor
+import ross as rs
+
+import bokeh.palettes as bp
+from bokeh.plotting import figure
+from bokeh.models import (
+        ColumnDataSource,
+        HoverTool,
+        Span,
+        Label,
+)
+
+# set bokeh palette of colors
+bokeh_colors = bp.RdGy[11]
+
+
+class Report:
+    def __init__(self, rotor, safety_factor=1.0):
```

Comment:
What is the use of `safety_factor`? Do we really need this? If so, we should explain in the docstrings.

Reply:
I thought about including this only for user's customization, but the Standards do not use safety factor for anything. I'm removing it.